src/control/cmd/daos/container.go

```diff
@@ -405,6 +406,25 @@ func (cmd *containerCreateCmd) contCreate() (string, error) {
 		contID = cmd.contUUID.String()
 	}
 
+	if len(cmd.Attrs.ParsedProps) != 0 {
```

Comment:
Given the discussion on the approach to the UNS API, this should be broken out into a helper function shared by both paths, right? Then the UNS version will additionally call `duns_link_cont`.

Reply:
I am going to refactor the code to try to have one function.

Reply:
Fixed with commit 819b856455ea7ad9a0fe3e3bb0d851aaa5b10afc